packages/plugin-client-drizzle/src/generate.ts

```diff
@@ -0,0 +1,65 @@
+import { BaseSchema, Schemas } from '@xata.io/client';
+import { boolean, date, decimal, integer, pgTable, text } from 'drizzle-orm/pg-core';
+import { exhaustiveCheck } from './utils';
+
+function buildColumnType(column: Schemas.Column) {
+  switch (column.type) {
+    case 'string':
+    case 'text':
+    case 'email':
+    case 'link':
+      return text(column.name);
+    case 'bool':
+      return boolean(column.name);
+    case 'int':
+      return integer(column.name);
+    case 'float':
+      return decimal(column.name);
+    case 'datetime':
+      return date(column.name);
+    case 'multiple':
+    case 'object':
+    case 'vector':
+    case 'file[]':
+    case 'file':
+    case 'json':
```

Comment:
file and json can be mapped to `jsonb`.

Reply:
I'll leave them as TODO, it's experimental and it's not clear how they'll map to the ORM